microsoft-edge/webview2/release-notes.md

```diff
@@ -245,7 +271,12 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
 
 ##### [.NET/C#](#tab/dotnetcsharp)
 
-n/a
+* `CoreWebView2CustomSchemeRegistration` Class:

```

Comment:
Are we adding information about the existing APIs for .NET and Win32?

Reply:
yes, because:

1. The documentation might be more complete in the Win32 and .NET Ref docs.  

2. The design of these API Ref tab-groups is to fill in all 3 tabs.  Ensures consistency and flexibility rather than omitting Win32 and .NET Ref links, which isn't the design for the 3-tab tab groups for API Ref for wv2.